src/Tests/dotnet-new.Tests/Approvals/DotnetCSharpClassTemplatesTest.struct.langVersion=9.0.targetFramework=netstandard2.0.verified/std-streams/stdout.txt

```diff
@@ -0,0 +1,3 @@
+﻿Warning: Failed to evaluate bind symbol 'ImplicitUsings', it will be skipped.
+Warning: Failed to evaluate bind symbol 'ImplicitUsings', it will be skipped.
```

Comment:
Just curious - is this warning expected and wanted? And is it expected to have it twice?
Just asking as this is something that users will see